include/RED4ext/ResourceLoader.hpp

```diff
@@ -86,23 +92,17 @@ struct ResourceToken
         return error;
     }
 
-    [[nodiscard]] inline operator T*() const noexcept
+    [[nodiscard]] inline operator T*() noexcept
     {
         return resource.GetPtr();
     }
 
-    struct Unk38
-    {
-        using AllocatorType = Memory::EngineAllocator;
-        uint8_t unk0[0x78];
-    };
-    RED4EXT_ASSERT_SIZE(Unk38, 0x78);
-
     WeakPtr<ResourceToken<T>> self;                    // 00
     DynArray<SharedPtr<ResourceToken<>>> dependencies; // 10
     SharedMutex lock;                                  // 20
     Handle<T> resource;                                // 28
-    SharedPtr<Unk38> unk38;                            // 38
+    void* unk38;                                       // 38 - SharedPtr<Unk38>.instance
```

Comment:
What is the reason to convert this to pointers instead of keeping the SharedPtr object?

Reply:
It needs to be decoded to properly destroy and free and I don't want to do that yet .) The only known dtor is for `SharedPtr<Unk38>` with inline dtor for `Unk38` so for now I call it instead.

Reply:
Ahhh, 👍